docs/faq.md

```diff
@@ -1,7 +1,7 @@
 # FAQ
 
 
-## Which libraries are available for Chrysalis?
+## Which Libraries are Available for Chrysalis?
 
 At this time, there are a handful of libraries available for Chrysalis which are outlined below:
 
```

Comment:
**For line 8**
- Please add a link to the iota.rs repository or documentation.

**For line 10**
- Please add a link to the wallet.rs repository or documentation.
- Is it coins or tokens?

Reply:
Coins, I believe!